dev/tasks/dev/utils/sanitize.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+module.exports = {
+	appendPeriodIfMissing( text ) {
+		text = text ? text.trim() : '';
```

Comment:
I'd keep defaulting to an empty string outside this function – in the `inquiries.js`. E.g. if we add more functions like this, we won't need to check the value in all of them. Plus, the documentation can be simpler – this function accepts a string and nothing more.